regression/goto-instrument/slice_general007/main.c

```diff
@@ -0,0 +1,78 @@
+// This is a benchmark for the full-slicer
+// This is a simplified version of end-to-end regression tests
+// 'general005', 'general006', and 'general007' of the security-scanner.
+
+#include <malloc.h>
```

Comment:
malloc.h is not portable. Yet you don't really seem to need it - maybe that should be stdlib.h?